main.go

```diff
@@ -19,11 +20,19 @@ import (
 	pb "gopkg.in/cheggaaa/pb.v1"
 )
 
+var (
+	cleanup      []func() error
+	cleanupMutex sync.Mutex
+)
+
 func main() {
+	// Do any cleanup on exit
+	defer doCleanup()
+
 	app := cli.NewApp()
 	app.Name = "ipget"
 	app.Usage = "Retrieve and save IPFS objects."
-	app.Version = "0.7.0"
+	app.Version = "0.5.0"
```

Comment:
?

Reply:
Fixed.  This change may have been made some time ago, and this file copied into a current branch. This goes away completely with https://github.com/ipfs/ipget/pull/86